### PR TITLE
[TS-351] All Search Results page

### PR DESCRIPTION
**Pull request by AlyaNaseer**
For now the data is hard coded but it has been set up for the back end. The "More information" button doesn't work

1. Login
2. Use search in navigation bar and search for anything
3. Hit Enter
4. You should see "Search results: _______" where _________ is what you typed in the search box
5. You should see "search/keyword=________?" in the URL

image

Here is a video demonstrating the functionality:
https://user-images.githubusercontent.com/46854444/159055814-707cd633-ad0f-4e7e-a25f-1a17c7cc5596.mp4

Here is a video demonstrating the responsiveness on different screen sizes (and modifications to responsiveness of Following/Followers pages):
https://user-images.githubusercontent.com/46854444/159064400-d635560c-eac3-4f39-b47f-a9e9a8de5c81.mp4

### CODE REVIEW

File: src/components/SearchStockBlock/StockBlock.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        xl="6"
+        class="text-left mx-0 mt-1 py-0 pr-0"
+      >
+        <v-list-item color="black">
+          <v-list-item-content class="py-1">
+            <v-list-item-title
+              class="text-caption text-wrap"
+              data-cy="nasdaq"
+            >
+              NASDAQ:
```
Comment (rileykc98) [inline src/components/SearchStockBlock/StockBlock.vue:39]: Would "ticker" be a more accurate label here? NASDAQ is one specific stock exchange platform, for example there's also stocks that trade on the TSX and NYSE. Unless we only display stocks that trads on NASDAQ. In which case please disregard.

Comment (AlyaNaseer) [reply src/components/SearchStockBlock/StockBlock.vue:39]: I actually didn't know this! I can change it to Ticker!

**Review by Ming424 — APPROVED** [review]
I would personally suggest the [More information _icon] button can be reduce into simply [_icon] and with more height, but you dont have to change it.
Look good to me!

**Comment by AlyaNaseer** [comment]
> I would personally suggest the [More information _icon] button can be reduce into simply [_icon] and with more height, but you dont have to change it. Look good to me!

That's a good suggestion, I just thought having the text makes it clearer to the user what is the expected behaviour when clicking the button

**Review by bowe99 — APPROVED** [review]

**Pull request merged**